src/map/script.hpp

```diff
@@ -2090,6 +2090,11 @@ enum e_convertpcinfo_type : uint8 {
 	CPC_ACCOUNT   = 2
 };
 
+enum e_instance_warpall_flag : uint8 {
```

Comment:
```suggestion
enum e_instance_warpall_flag{
```
To fix the issue mentioned by @vstumpf